lua/telescope/_extensions/file_browser/make_entry.lua

```diff
@@ -159,6 +161,10 @@ local make_entry = function(opts)
       end
     end
 
+    if is_symlink then
```

Comment:
nit: I think we can just do `entry.lstat.type == "link"` here directly since we only check that once here anyways